query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryRequestUtil.java

```diff
@@ -72,29 +77,65 @@
         .build();
   }
 
-  public static boolean isDateTimeFunction(Expression expression) {
-    return expression.getValueCase() == ValueCase.FUNCTION
-        && expression.getFunction().getFunctionName().equals("dateTimeConvert");
+  public static Filter createContainsKeyFilter(AttributeExpression complexAttributeExpression) {
+    return createContainsKeyFilter(
+        complexAttributeExpression.getAttributeId(), complexAttributeExpression.getSubpath());
   }
 
-  public static boolean isComplexAttribute(Expression expression) {
-    return expression.getValueCase().equals(ATTRIBUTE_EXPRESSION)
-        && expression.getAttributeExpression().hasSubpath();
+  public static Filter createContainsKeyFilter(String column, String value) {
```

Comment:
nit: can be inline above function or can be private as it (`createContainsKeyFilter(String column, String value)` is only used in Tests now. But, can be taken as follow up.